### PR TITLE
Gmiguel

### PR DESCRIPTION
**Pull request by gracemiguel**
adding us02, us03, us02_test and us03_test, still having an import issue for tests

### CODE REVIEW

File: src/us03.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import copy
+import numpy as np
+import pandas as pd
+import datetime
+import os
+sys.path.append(os.path.abspath('../src/seeds'))
+import Project02
+
+#user03 checks to see if birth dates are before death dates.
+def user03(gedcom_file):
```
Comment (ming1in) [inline src/us03.py]: nit: function name should match file name

File: test/us02_test.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import sys
+import unittest
+import os
+sys.path.append(os.path.abspath('../src'))
+os.chdir(os.path.abspath('../test/seeds'))
+import us02
+
+
+class Testuser02(unittest.TestCase):
```
Comment (ming1in) [inline test/us02_test.py]: nit: function name should match the file name

File: test/us03_test.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import unittest
+import sys
+import os
+sys.path.append(os.path.abspath('../src'))
+os.chdir(os.path.abspath('../test/seeds'))
+import us03
+
+class TestUser03(unittest.TestCase):
```
Comment (ming1in) [inline test/us03_test.py]: nit: function name should match file name

File: src/us02.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+'''
+import sys
+import os
+import pandas as pd 
+import Project02
+from datetime import datetime
+import copy
+
+#this function is to determine if the marriage date is after the birth date
+def user02(gedcom_file):
```
Comment (ming1in) [inline src/us02.py]: nit: function name should match file name

**Review by ming1in — APPROVED** [review]
Nice! just some small naming stuff

**Review by samanthainneo99 — APPROVED** [review]
I'm approving this so it can be merged in

File: src/us03.py (hunk 1 above)
Comment (gracemiguel) [reply src/us03.py]: changed the name

File: src/us02.py (hunk 4 above)
Comment (gracemiguel) [reply src/us02.py]: changed name to file name

File: test/us03_test.py (hunk 3 above)
Comment (gracemiguel) [reply test/us03_test.py]: fixed

**Review by arana23 — APPROVED** [review]
incrediblé

**Pull request merged**